region/maridia/outer/Mama Turtle Room.json

```diff
@@ -886,6 +933,52 @@
       ],
       "flashSuitChecked": true,
       "note": "To save energy, start from the upper right ledge; it may require repositioning Mama Turtle to the right to get to the ledge."
+    },
+    {
+      "link": [5, 6],
+      "name": "Base",
+      "requires": [
+        "canUseGrapple",
+        "SpaceJump"
+      ]
+    },
+    {
+      "link": [6, 3],
+      "name": "Base",
+      "requires": []
+    },
+    {
+      "link": [6, 5],
+      "name": "Base",
+      "requires": [
+        {"and": [
+          "canUseGrapple",
```

Comment:
I'd like for these to be preciseGrapples, from the side walls.
HiJump with a running jump gets you close enough to comfortably grapple the block, but it isn't clear how to run on the slope safely.

Reply:
Fair enough!
```json
    {
      "link": [6, 5],
      "name": "Base",
      "requires": [
        "canPreciseGrapple",
        "SpaceJump"
      ]
    }
```